src/client/lobby/lobby.js

```diff
@@ -16,11 +19,27 @@ export class Lobby extends Component {
         }
     }
 
+    handleChange(event) {
+        cloakService.messageSetUsername(event.target.value);
+        storageService.storeName(event.target.value);
+    }
+
+    getUsername() {
+        return storageService.getUser().name;
+    }
+
     render() {
         return (
             <div>
                 <div className="text-center">
-                    <h1>Lobby</h1>
+                    <h1>
+                        Lobby
+                        <label className={style['set-username-label']}>
+                            <label>{'Username: '}</label> 
```

Comment:
The point being made: remove the braces and just put `Username: ` in the label :smile: